test-data/gencode.v29.annotation.bed

```diff
@@ -1,35 +1,35 @@
-chr1	11868	14409	gene	0	+
+chr1	11868	14409	gene	0	+	11868	14409	0	1	2541	0
```

Comment:
Wondering if the gene name should be in the 4th column?

Reply:
> Wondering if the gene name should be in the 4th column?

Yes, that's indeed a bug in the original implementation, see my other comment.